archetype-common/src/main/java/org/apache/maven/archetype/common/util/PomUtils.java

```diff
@@ -127,7 +129,16 @@ public static boolean addNewModule( String artifactId, Reader fileReader, Writer
             tf.setAttribute( XMLConstants.ACCESS_EXTERNAL_STYLESHEET, "" );
 
             tf.setAttribute( "indent-number", 2 );
-            Transformer tr = tf.newTransformer();
+            final Transformer tr;
```

Comment:
This belongs inside the try block

Reply:
> This belongs inside the try block

Hoｗ to fix?

Reply:
initialize and declare on the same line; make the try block bigger

Reply:
014ca12b ?